### PR TITLE
feat(prisma): add platform api calls for apikey cmds

### PR DESCRIPTION
**Pull request by jasonkuhrt**
This PR brings Platform API calls from `apikey` Prisma CLI commands.

### CODE REVIEW

File: packages/cli/src/platform/apikey/create.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  public async parse(argv: string[]) {
+    const args = arg(argv, {
+      ...platformParameters.project,
+      '--display-name': String,
+      '-d': '--display-name',
+    })
+    if (isError(args)) return args
+    const token = getRequiredParameter(args, ['--token', '-t'], 'PRISMA_TOKEN')
```
Comment (janpio) [inline packages/cli/src/platform/apikey/create.ts:22]: Aside: This env var name should probably be more PDP specific.

Comment (jasonkuhrt) [reply packages/cli/src/platform/apikey/create.ts:22]: I also think its suffix should match the flag name, whatever it changes to.

@luanvdw this is the token I was mentioning before.

Based on some other ongoing discussions IIUC the most accurate term here would be `PRISMA_PLATFORM_MANAGEMENT_TOKEN` (but this is not settled yet).

File: packages/cli/src/platform/accelerate/disable.ts [hunk 2]
```diff
@@ -26,6 +26,6 @@ export class Disable implements Command {
         intent: 'disable',
         projectId: project,
       },
-    })
+    }) as Promise<any>
```
Comment (jasonkuhrt) [inline packages/cli/src/platform/accelerate/disable.ts:29]: These have been added to make what was previously implicitly this type explicit.

It needs to be fixed later.

Right now the local CLI framework here only allows strings to be returned.

Changing this requires increasing the scope of this PR and I think we can defer that.

The casts here are not changing behaviour, so, its still progress, by making them explicit, in this PR.

Comment (sampolahtinen) [reply packages/cli/src/platform/accelerate/disable.ts:29]: Let's add some of your remarks as TODO comments?

Comment (jasonkuhrt) [reply packages/cli/src/platform/accelerate/disable.ts:29]: There's a bunch of callsites though and in this case the `as` casting serves as kind of todo anyways. But feel free to push a commit if that helps!

**Review by Jolg42 — APPROVED** [review]
Looks ok, though without tests 😉

File: packages/cli/src/utils/platform.ts [hunk 3]
```diff
@@ -71,7 +71,7 @@ export const platformRequestOrThrow = async (params: {
   token: string
   path: string
   payload?: object
-}) => {
+}): Promise<object> => {
```
Comment (sampolahtinen) [inline packages/cli/src/utils/platform.ts:74]: @jasonkuhrt lets re-iterate on this one. I think we could just remove explicit return type and avoid casting `as Promise<any>`

I will approve and merge this PR as it is.

**Review by sampolahtinen — APPROVED** [review]

**Pull request merged**